app/src/main/res/values-ko/strings.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Shortcut strings -->
+    <string name="scut_deleted_pc">PC 삭제됨</string>
+    <string name="scut_not_paired">PC와 페어링되지 않음</string>
+
+    <!-- Help strings -->
+    <string name="help_loading_title">도움말 뷰어</string>
+    <string name="help_loading_msg">도움말 페이지 로딩중…</string>
+
+    <!-- PC view menu entries -->
+    <string name="pcview_menu_app_list">게임 리스트 보이기</string>
+    <string name="pcview_menu_pair_pc">PC와 페어링</string>
+    <string name="pcview_menu_unpair_pc">페어링 해제</string>
+    <string name="pcview_menu_send_wol">Wake-On-LAN request 보내기</string>
+    <string name="pcview_menu_delete_pc">PC 삭제</string>
+
+    <!-- Pair messages -->
+    <string name="pairing">페어링 중…</string>
+    <string name="pair_pc_offline">컴퓨터가 오프라인입니다.</string>
+    <string name="pair_pc_ingame">컴퓨터가 아직 게임을 실행중입니다. 페어링 전 게임을 종료하세요.</string>
+    <string name="pair_pairing_title">페어링</string>
+    <string name="pair_pairing_msg">아래 PIN을 페어링 할 PC에 입력하세요:</string>
+    <string name="pair_incorrect_pin">PIN이 올바르지 않음</string>
+    <string name="pair_fail">페어링 실패</string>
+    <string name="pair_already_in_progress">페어링이 이미 진행중입니다</string>
+
+    <!-- WOL messages -->
+    <string name="wol_pc_online">컴퓨터가 온라인 상태입니다</string>
+    <string name="wol_no_mac">Geforce Experience가 MAC 주소를 보내지 않아 PC를 깨울 수 없습니다.</string>
+    <string name="wol_waking_pc">PC를 깨우는 중…</string>
+    <string name="wol_waking_msg">PC를 깨우는 데에 약간의 시간이 걸립니다.
+        PC가 깨워지지 않으면, Wake-On-Lan이 설정되어있는지 확인하세요.
+    </string>
+    <string name="wol_fail">Wake-On-LAN 패킷을 보내는 데에 실패했습니다</string>
+
+    <!-- Unpair messages -->
+    <string name="unpairing">페어링 해제중…</string>
+    <string name="unpair_success">페어링 해제 완료</string>
+    <string name="unpair_fail">페어링 해제 실패</string>
+    <string name="unpair_error">장치가 페어링되지 않음</string>
+
+    <!-- Errors -->
+    <string name="error_pc_offline">컴퓨터가 오프라인 상태입니다</string>
+    <string name="error_manager_not_running">ComputerManager 서비스가 실행되지 않고 있습니다. 몇 초 기다리거나 앱을 다시 시작하세요.</string>
+    <string name="error_unknown_host">호스트 응답에 실패함</string>
+    <string name="error_404">GFE가 HTTP 404에러를 보냈습니다. 지원되는 GPU가 설치되어 있는지 확인하세요.
+	    원격 데스트톱 응용 프로그램을 사용하는 것도 이 에러를 유발할 수 있습니다. PC를 다시 시작하거나 Geforce Experience를 다시 설치해보세요. 
+    </string>
+
+    <!-- Start application messages -->
+    <string name="conn_establishing_title">연결 수립중</string>
+    <string name="conn_establishing_msg">연결 시작</string>
+    <string name="conn_metered">경고: Your active network connection is metered!</string>
```

Comment:
Metered Connection means that you're in mobile network and every MBs you use the mobile phone operator will ask you to pay for this